src/api/app/controllers/source_project_package_meta_controller.rb

```diff
@@ -1,7 +1,26 @@
 class SourceProjectPackageMetaController < SourceController
   before_action :require_package_name, only: [:show, :update]
+  before_action :set_rdata, only: [:update]
+  before_action :validate_project_name, only: [:update]
+  before_action :validate_package_name, only: [:update]
   validate_action update: { request: :package, response: :status }
 
+  def set_rdata
```

Comment:
do we really need a filter for that? In my opinion it is offuscating the code... the name of the method doesn't really tell me what is doing and it is just one line.... :thinking: And to do this you need to define an instance variable that is only used in the method itself.... :unamused:

Reply:
the method says exactly what it's doing. set rdata. Technically it's not a filter, since it's not filtering anything and after more refactoring in others source*_controllers it will probably end-up being moved to the parent class, as ```before_action``` where it can be called before others similar actions. If you have any suggestion for a better name, just say it.